### ISSUE TITLE
Rise from the grave

### ISSUE THREAD

**Issue by fretka46** [grufkork/scpmapper#1]
I really like this idea, I've been experimenting with it for a while and it's really well done. Is there any way to incorporate actual maps into this?

**Comment by grufkork** [comment]
Sorry, not quite sure I understand. Do you mean actual graphical maps? Shouldn't be too difficult, find some rendering backend and just map the different pathway characters to images.

Does the tool still work? I thought the maps had changed a LOT since I last worked on it...

**Comment by fretka46** [comment]
Yes, the maps have changed. New rooms have been added making the zones bigger and their generation changed. Anyway, the generation of rooms in a zone is still not random and there are a handful of possible combinations that are used in the game each round.

**Comment by grufkork** [comment]
Well, shouldn't be more difficult than updating the map files in scp-sl-layouts and layouts.txt. I don't have much interest in this currently, but I'll help if you want to have a stab at it!

**Comment by kellentow** [comment]
Hey I know this is a old issue but I did fork this and updated all the light maps, the rest should be done soon but I do have finals so it might take a week

**Comment by grufkork** [comment]
Nice, just PR if you got anything (or make your fork the main one)! As previously said I don't care much for this project, but I'll merge any good changes and your stuff looks promising 👍
